src/main/scala/dal/table/blockDetailTable.scala

```diff
@@ -0,0 +1,17 @@
+package dal.table
+
+import sqlest._
+
+class BlockDetailTable(alias: Option[String]) extends Table(block_detail) {
+
+  val blockID = column[Int]("block_id")
```

Comment:
Once the domain has been completed, I'd recommend revisiting the column types. We might have a BlockId class for example (a simple wrapper for Int), and so the column type of this blockId column would be BlockId rather than Int.

Reply:
Wise, and adds scalability should we want to add anything to a "Block ID". Would those classes go in the dal folder?